astro/src/components/api/DeprecatedSince.astro

```diff
@@ -1,8 +1,20 @@
 ---
 interface Props {
-  since: string;
+  inline: boolean,
+  since: string
 }
 
-const { since }: Props = Astro.props;
+const { inline, since }: Props = Astro.props;
+const hasContent = Astro.slots.has("default");
+let classList = 'text-red-700 italic';
+if (inline) {
```

Comment:
This is confusing to me. The span is inline by default, it looks like what you are going for is just to minimize the margin between the next line of text. I don't think we need a new div for that. Or is this meant to actually encapsulate the content in a meaningful way? Should it be an aside?